src/Serializer/AbstractItemNormalizer.php

```diff
@@ -65,6 +67,12 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
         $this->setCircularReferenceHandler(function ($object) {
             return $this->iriConverter->getIriFromItem($object);
         });
+
+        if (null === $identifiersExtractor) {
+            @trigger_error(sprintf('Not passing an instance of IdentifiersExtractorInterface in "%s" is deprecated and will be mandatory in version 3.0. Not passing it is deprecated since 2.2.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
`2.3` nay ? Kinda strange to introduce a deprecated on > 2.2.0...

Reply:
Not really, not passing an `id` in a POST request is a bug to me but I don't care as long as it's published soon it can also go into 2.3 😄 Deprecating in bugfix releases is fine.